src/meta/types.ts

```diff
@@ -457,8 +457,10 @@ export interface OpenableOptions {
 	aliases: string[];
 }
 
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface OpenableOpenOptions {}
+export interface OpenableOpenOptions {
+	lvl?: number;
+	size?: 'big' | 'small';
```

Comment:
either declare this as just a string or create a type that only accepts `'big' | 'small'` and use that here